components/de_amp/browser/de_amp_util.cc

```diff
@@ -7,6 +7,7 @@
 
 #include "base/feature_list.h"
 #include "base/no_destructor.h"
+#include "base/types/expected.h"
```

Comment:
you already have this one in the header

Reply:
I thought we were supposed to always include what we use? @bridiver

Reply:
then you need to include <string> and <gurl.h> :) 
technically there is no need to duplicate includes between corresponding .h/.cc